Auth0/CredentialsManager.swift

```diff
@@ -194,40 +195,14 @@ public struct CredentialsManager {
     }
 
     func hasExpired(_ credentials: Credentials) -> Bool {
-
         if let expiresIn = credentials.expiresIn {
             if expiresIn < Date() { return true }
         }
 
-        if let token = credentials.idToken,
-            let tokenDecoded = decodeJwt(token),
-            let exp = tokenDecoded["exp"] as? Double {
-            if Date(timeIntervalSince1970: exp) < Date() { return true }
+        if let token = credentials.idToken, let jwt = try? decode(jwt: token) {
+            return jwt.expired
```

Comment:
See the implementation in https://github.com/auth0/JWTDecode.swift/blob/master/JWTDecode/JWTDecode.swift#L66.